pocket_coffea/parameters/executor_options_defaults.yaml

```diff
@@ -44,6 +44,7 @@ parsl-condor@DESY_NAF:
   logs-dir: logs_parsl
   queue: ""
   walltime: "12:00:00"
+  ignore-grid-certificate: false
```

Comment:
this has been now added to the general defaults.  But it is not harmful to keep it also in the defaults by-sites if you wish